ballerina-command/src/main/java/org/ballerinalang/command/cmd/FetchCommand.java

```diff
@@ -49,13 +49,32 @@ public void execute() {
             return;
         }
 
-        if (fetchCommands == null) {
+        if (fetchCommands == null || fetchCommands.size() == 0) {
```

Comment:
We thought not to include fetch command for this release. Shall we remove it? 

We can keep this class, but remove where this is instantiated and mentioned in help text, etc.

Reply:
Lets remove from the main method and keep these logics

Reply:
Will remove those in next PR.